### PR TITLE
w1-d3 View and View Group

### PR DESCRIPTION
**Pull request by at-hungtran2**
#1

### CODE REVIEW

**Comment by at-hungtran2** [comment]
please review it, thanks.

File: app/src/at-hungtran/res/layout/activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <Button
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:id="@+id/idBtn"
+        android:text="@string/viewgroup"
+        android:layout_centerHorizontal="true"/>
+</RelativeLayout>
```
Comment (at-chauhoang) [inline app/src/at-hungtran/res/layout/activity_main.xml]: Please add break line and format code and follow convention doc.

File: app/src/main/res/drawable/border.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <solid
+        android:color="#22ef2d" >
+    </solid>
+
+    <!-- và đây là bán kính đường tròn ở 4 góc -->
+    <corners
+        android:radius="50dp" >
+    </corners>
+
+</shape>
```
Comment (at-chauhoang) [inline app/src/main/res/drawable/border.xml]: Remove it, this is main package.

File: app/src/main/res/drawable/border1.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <solid
+        android:color="#1659df" >
+    </solid>
+
+    <!-- và đây là bán kính đường tròn ở 4 góc -->
+    <corners
+        android:radius="50dp" >
+    </corners>
+
+</shape>
```
Comment (at-chauhoang) [inline app/src/main/res/drawable/border1.xml]: Remove it, this is main package.

File: app/src/main/res/layout/activity_layoutfb.xml [hunk 4]
```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (at-chauhoang) [inline app/src/main/res/layout/activity_layoutfb.xml]: Remove it, this is main package.

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
```
Comment (at-chauhoang) [inline app/src/main/res/layout/activity_main.xml]: Remove it, this is main package.

File: app/src/main/res/layout/single_item.xml [hunk 6]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (at-chauhoang) [inline app/src/main/res/layout/single_item.xml]: Remove it, this is main package.

File: app/src/at-hungtran/res/layout/single_item.xml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/textView"
+        android:gravity="center"
+        android:layout_marginLeft="20dp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="20dp"
+        app:fontFamily="Arial"
```
Comment (at-chauhoang) [inline app/src/at-hungtran/res/layout/single_item.xml]: Use: android:fontFamily="Arial"

File: app/src/at-hungtran/res/layout/single_item.xml [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:padding="8dp"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/textView"
+        android:gravity="center"
```
Comment (at-chauhoang) [inline app/src/at-hungtran/res/layout/single_item.xml]: Remove this attribute.

File: app/src/at-hungtran/res/layout/single_item.xml [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+            android:gravity="center"
+            android:paddingLeft="10dp"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor="#2b67e9"
+            android:paddingStart="10dp"
+            tools:ignore="RtlSymmetry" />
+    </LinearLayout>
+
+
```
Comment (at-chauhoang) [inline app/src/at-hungtran/res/layout/single_item.xml]: Remove this line.

File: app/src/at-hungtran/res/layout/single_item.xml [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+</LinearLayout>
```
Comment (at-chauhoang) [inline app/src/at-hungtran/res/layout/single_item.xml]: Please add break line and format code and set name by convention doc

**Pull request merged**